src/templates/Blog/blog.jsx

```diff
@@ -2,7 +2,9 @@ import React from "react"
 import { Layout } from "../../components/Layout"
 import { SEO } from "../../components/SEO"
 import { graphql } from "gatsby"
+import helpers from "../../styles/helpers.module.css"
 import styles from "./blog.module.css"
+import { CTABlock } from "../../components/CTABlock"
```

Comment:
Add this to after the SEO one to group all the component imports and style imports